test/embedded/builtin-float.swift

```diff
@@ -2,7 +2,7 @@
 // RUN: mkdir -p %t/include
 // RUN: %{python} %utils/split_file.py -o %t %s
 
-// RUN: %target-swift-frontend -target armv7em-none-none-eabi -emit-ir %t/Main.swift -enable-experimental-feature Embedded -module-cache-path %t/ModuleCache -Xcc -I%t/include
+// RUN: %target-swift-frontend -target %target-triple -emit-ir %t/Main.swift -enable-experimental-feature Embedded -module-cache-path %t/ModuleCache -Xcc -I%t/include
```

Comment:
let's drop this change -- this would stop testing this interesting scenario (there is no dedicated CI job for the armv7em-none-none-eabi triple)